java/src/main/c++/jni_spark_vw_generated.h

```diff
@@ -0,0 +1,211 @@
+/* DO NOT EDIT THIS FILE - it is machine generated */
```

Comment:
How is this file generated?

Reply:
using the cmake JNI thingy above.

Reply:
added comment in the CMakeFile.txt under Development